saleor/graphql/checkout/mutations/checkout_lines_update.py

```diff
@@ -221,3 +224,30 @@ def _get_variants_from_lines_input(cls, lines: list[dict]) -> list[ProductVarian
     @classmethod
     def _get_grouped_lines_data(cls, lines, existing_lines_info):
         return group_lines_input_data_on_update(lines, existing_lines_info)
+
+    @classmethod
+    def _validate_gift_line(
```

Comment:
Should we allow updating the gift line? I mean, maybe we shouldn't check the quantity and return a validation error when somebody tries to update the gift line?